test/e2e/apimachinery/initializers.go

```diff
@@ -262,6 +262,29 @@ var _ = SIGDescribe("Initializers", func() {
 		Expect(err).NotTo(HaveOccurred())
 		Expect(len(pods.Items)).Should(Equal(1))
 	})
+
+	It("will be set to nil if a patch removes the last pending initializer", func() {
```

Comment:
@caesarxuchao @smarterclayton Use of the initializers field should be feature gated and disabled/cleared by default, and these tests should be marked with a Feature:Initializers indicator so that only e2e setups that enable the alpha initializers feature run these.

Reply:
opened https://github.com/kubernetes/kubernetes/pull/51436

Reply:
This particular PR shouldn't require the initializer admission controller or the intializer configuration API being enabled. I'll take a look at kops failure.

Reply:
@liggitt was right. The mutation is done by the initializer admission controller so this test does need to be feature gated.